uint16[] sw
uint16[] error
uint8[] opMode
uint8[] complement
float64[] q
float64[] qd
float64[] cur
float64[] dc_v